MarsRoverTest/MarsRoverTest.UnitTest/RoverTests/WhenTurningRight.cs

```diff
@@ -0,0 +1,82 @@
+﻿using MarsRover;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MarsRoverTest.UnitTest.RoverTests
+{
+    [TestClass]
+    public class WhenTurningRight
+    {
+        [TestMethod]
+        public void AndFacingNorthThenRoverFacesEast()
+        {
+            //arrange
+            var rover = new Rover { Direction = Direction.North };
+            //act
+            rover.TurnRight();
+            //assert
+            Assert.AreEqual(Direction.East, rover.Direction);
+        }
+
+        [TestMethod]
+        public void AndFacingEastThenRoverFacesSouth()
+        {
+            //arrange
+            var rover = new Rover { Direction = Direction.East };
+            //act
+            rover.TurnRight();
+            //assert
+            Assert.AreEqual(Direction.South, rover.Direction);
+        }
+
+        [TestMethod]
+        public void AndFacingSouthThenRoverFacesWest()
+        {
+            //arrange
+            var rover = new Rover { Direction = Direction.South };
+            //act
+            rover.TurnRight();
+            //assert
+            Assert.AreEqual(Direction.West, rover.Direction);
+        }
+
+        [TestMethod]
+        public void AndFacingWestThenRoverFacesNorth()
+        {
+            //arrange
+            var rover = new Rover { Direction = Direction.West };
+            //act
+            rover.TurnRight();
+            //assert
+            Assert.AreEqual(Direction.North, rover.Direction);
+        }
+
+        [DataTestMethod]
+        [DataRow(DisplayName = "AndFacingNorthThenTheRoverFacesWest")]
+        [DataRow(DisplayName = "AndFacingEastThenTheRoverFacesNorth")]
+        [DataRow(DisplayName = "AndFacingSouthThenRoverFacesEast")]
+        [DataRow(DisplayName = "AndFacingWestThenRoverFacesSouth")]
+        public void AndRoverFacesInitialDirectionThenRoverFacesFinalDirection(Direction initialDirection, Direction finalDirection)
```

Comment:
So out of curiosity, do these tests pass on your machine? Because we're not passing in any values for `initialDirection` or `finalDirection`?

Reply:
Neither test was passing on my end as well, I was going to ask for a hint towards the syntax we wanted but this statement is all I need! I will update it and get that fixed.